cmd/juju/application/bundle/bundle_test.go

```diff
@@ -508,3 +532,68 @@ relations:
 - - wordpress:db
   - mysql:db
 `
+
+const mixedSeriesBaseBundle = `
+series: bionic
+default-base: ubuntu@18.04
+applications:
+  mysql:
+    charm: ch:mysql
+    revision: 42
+    channel: stable
+    series: xenial
+    base: ubuntu@16.04
+    num_units: 1
+    to:
+    - "0"
+  wordpress:
+    charm: ch:wordpress
+    channel: stable
+    revision: 47
+    series: jammy
+    num_units: 1
+    to:
+    - "1"
+machines:
+  "0":
+    series: xenial
+    base: ubuntu@16.04
```

Comment:
Please do not add more instances of using unsupported series, unless that is the purpose of the tests. Given the name of the const, that is not clear.

Reply:
I believe this is a carry over from the bundle this was copies from. Happy to change

Reply:
please update the entire bundle, and mixedSeriesBaseBundleMismatch

Reply:
Have done